Source/Extensions/Blazorise.DataGrid/_DataGridDetailRow.razor

```diff
@@ -1,6 +1,6 @@
 ﻿@typeparam TItem
 @inherits _BaseDataGridDetailRow<TItem>
-<TableRow>
+<TableRow style="background-color: unset;">
```

Comment:
This one is a bit tricky if we want it to be abstracted. Let's leave it as it is and I can get beck to it sometimes later. So for now it's OK.

Reply:
So leave the `unset` or remove it? 😄 

Yeah, I guess it would depend on the style provider used. 🤔 
This is probably more of a hotfix to remove the annoying flashing when hovering a "detail row", than a real fix.

Reply:
Just leave it for now. I will do it sometimes later. Just open new issue so I can keep track of it and forget about it.